dashboard-for-web/articles/web-dashboard-designer-mode/provide-data/manage-data-sources.md

```diff
@@ -21,7 +21,7 @@ The window allows you to do the following:
 | Action | Description |
```

Comment:
Line 11: To add a new data source, click **Add** next to the **Used Data Sources** list.

Can we update this to the following?

`Click **Add** next to the **Used Data Sources** list to add a new data source.`